packages/flutter/lib/src/material/icons.dart

```diff
@@ -2,8 +2,83 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/foundation.dart' show defaultTargetPlatform;
 import 'package:flutter/widgets.dart';
 
+// ignore_for_file: non_constant_identifier_names
+class _PlatformAdaptiveIcons {
+  static final bool isApplePlatform = _isApplePlatform();
+
+  static bool _isApplePlatform() {
```

Comment:
nit: there is precedent to maybe call this `isCupertino`

https://github.com/flutter/flutter/blob/84f3d28555/packages/flutter/lib/src/material/switch.dart#L407

Reply:
Done